packages/core/src/browser/keybinding.ts

```diff
@@ -463,3 +541,38 @@ export class KeybindingRegistry {
         }
     }
 }
+
+export namespace KeybindingRegistry {
+    export class KeybindingsResult {
```

Comment:
As mentionned above, I think it would be better to put that in a separate class, I don't see the need for a namespace here @akosyakov ?

Reply:
I was advised to do so in a previous review.